object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -162,23 +161,22 @@ private ReturnsRcvrAnnotatedTypeFactory getReturnsRcvrAnnotatedTypeFactory() {
    * <p>Package-private to permit calls from {@link ObjectConstructionTransfer}.
    */
   boolean returnsThis(final MethodInvocationTree tree) {
-    ReturnsRcvrAnnotatedTypeFactory rrATF = getReturnsRcvrAnnotatedTypeFactory();
+    ReturnsReceiverAnnotatedTypeFactory rrATF = getReturnsRcvrAnnotatedTypeFactory();
     ExecutableElement methodEle = TreeUtils.elementFromUse(tree);
     AnnotatedTypeMirror methodATm = rrATF.getAnnotatedType(methodEle);
     AnnotatedTypeMirror rrType =
         ((AnnotatedTypeMirror.AnnotatedExecutableType) methodATm).getReturnType();
-    return (rrType != null && rrType.hasAnnotation(This.class))
-        || hasOldReturnsReceiverAnnotation(tree);
+    return (rrType != null && rrType.hasAnnotation(This.class));
   }
 
-  /**
-   * Continue to trust but not check the old {@link
-   * org.checkerframework.checker.builder.qual.ReturnsReceiver} annotation, for
-   * backwards-compatibility.
-   */
-  private boolean hasOldReturnsReceiverAnnotation(MethodInvocationTree tree) {
-    return this.getDeclAnnotation(TreeUtils.elementFromUse(tree), ReturnsReceiver.class) != null;
-  }
+//  /**
```

Comment:
Remove, don't comment out

Reply:
Actually it was my mistake commenting this part, this method is no longer commented.